src/component/travel-form/travel-form.js

```diff
@@ -0,0 +1,56 @@
+// FOR LAB 32 
```

Comment:
Remove commented out files not related to lab assignment (fine to use for reference while working on lab, but be sure to remove them before submitting)